packages/language-support/src/helpers.ts

```diff
@@ -139,6 +110,80 @@ export function inRelationshipType(stopNode: ParserRuleContext) {
   return isDefined(relPattern);
 }
 
+class MyTokenStream extends CommonTokenStream {
+  tokens: antlrDefaultExport.Token[] = [];
+
+  constructor(lexer: CypherLexer, tokens: Token[]) {
+    super(lexer);
+    this.tokens = [...tokens];
+  }
+}
+
+export function findCaret(
```

Comment:
Just moves and reshapes this from the signature help, hopefully it can be reused for the auto-completion

Reply:
I feel like there should be a good way to find where the cursor is in the tree without having to go looking for it, but I've not found anything either, a bit dissapointing